lib/cmd/fh3/admin/environments/create.js

```diff
@@ -1,12 +1,18 @@
-module.exports = { 
+var log = require("../../../../utils/log");
+
+module.exports = {
   'desc' : 'Creates an environment.',
-  'examples' : [{ cmd : 'fhc admin environments create --id=<environment id> --label=<label> --targets=<mbaasTargetId1>,<mbaasTargetId2>', desc : 'Creates an environment'}],
+  'examples' : [{ cmd : 'fhc admin environments create --id=<environment id> --label=<label> --targets=<mbaasTargetId1>,<mbaasTargetId2>',
+    desc : 'Creates an environment'}],
   'demand' : ['id', 'label', 'targets'],
   'alias' : {},
   'describe' : {
     'id' : 'Unique identifier for your environment',
     'label' : 'A label describing your environment',
-    'targets' : 'Comma separated list of mBaaS Target hostnames'
+    'targets' : 'Comma separated list of mBaaS Target hostnames',
+    'autoDeployOnCreate' : 'option used to setup automatic deployment to this environment when creating new project/app.',
+    'autoDeployOnUpdate' : 'option used to setup automatic deployment to this environment when making source code changes within app.',
+    'noAppend' : 'option used to create environment without adding it to the list of environments available for the domain.'
```

Comment:
Given that this should not be really used by normal developers, can we add something like "do not use this flag unless you know what you are doing" to make it clear they should not really use it.